manifest: Updated Matter revision to 1.3 version tag (#15075)
nrfconnect/sdk-nrf: doc/nrf/protocols/matter/getting_started/low_power_configuration.rst

Short idle time and long idle time devices
==========================================

The Matter specification divides ICD into short idle time (SIT) and long idle time (LIT) devices.
The division is based on the Slow Polling Interval parameter: equal to or shorter than 15 seconds for SIT, and longer than 15 seconds for LIT.

> The typical use case for a SIT device are actuators, so the devices like door locks or window coverings, where the controller usually initiaties communication and the accessory device is expected to answer with a small latency.

Reviewer: ```suggestion
The typical use case for a SIT device are actuators, meaning devices such as door locks or window coverings, where the controller usually initiates communication and the accessory device is expected to answer with a small latency.
```

> The LIT devices are designed to be used for a sensors or light switches, so the devices that only report data and are not controllable.

Reviewer: ```suggestion
Conversely, LIT devices are designed to be used for sensors or light switches, devices that only report data and are not controllable.
```

> In such scenarios, the LIT device initiates communication and it is not able to answer with a small latency, but it can sleep for long hours and achieve much lower power consumption than an SIT.

Reviewer: ```suggestion
In such scenarios, the LIT device initiates communication and it is not able to answer with a small latency, but it can sleep for extended periods of time and achieve much lower power consumption than an SIT.
```

> In such scenario, the LIT device initiates communication and it is not able to answer with a small latency, but on the other hand it can sleep for a long hours and achieve much lower power consumption than SIT.

Reviewer: ```suggestion
In such scenarios, the LIT device initiates communication and it is not able to answer with a small latency, but it can sleep for long hours and achieve much lower power consumption than an SIT.
```

> The LIT device implementation requires multiple new features, such as Check-In protocol (CIP) support, ICD client registration and User Active Mode Trigger (UAT).

Reviewer: ```suggestion
The LIT device implementation requires multiple new features, such as Check-In protocol (CIP) support, ICD client registration, and User Active Mode Trigger (UAT).
```

> Despite the LIT mandatory requirements, the SIT device can also optionally enable mentioned features.

Reviewer: ```suggestion
These features are not required for SIT device implementation, but can be optionally enabled.
```

To configure the LIT, CIP or UAT, use the following Kconfig options:

* :kconfig:option:`CONFIG_CHIP_ICD_LIT_SUPPORT` to enable the Long Idle Time device support.
* :kconfig:option:`CONFIG_CHIP_ICD_CHECK_IN_SUPPORT` to enable the Check-In protocol support.

>   The Check-In protocol provides a way for a accessory device (ICD server) to notify the registered controller (ICD client) that it is available for communication.

Reviewer: ```suggestion
  The Check-In protocol provides a way for an accessory device (ICD server) to notify the registered controller (ICD client) that it is available for communication.
```

  This option is by default enabled for the LIT device.

> * :kconfig:option:`CONFIG_CHIP_ICD_CLIENTS_PER_FABRIC` to set the number of ICD clients that can be registered to a ICD server and benefit from Check-In protocol functionality.

Reviewer: ```suggestion
* :kconfig:option:`CONFIG_CHIP_ICD_CLIENTS_PER_FABRIC` to set the number of ICD clients that can be registered to an ICD server and benefit from CIP functionality.
```

* :kconfig:option:`CONFIG_CHIP_ICD_UAT_SUPPORT` to enable the User Active Mode Trigger support.

>   The User Active Mode Trigger allows to trigger the ICD device to move from the idle to active state and make it immediately responsive, for example to change its configuration.

Reviewer: ```suggestion
  The User Active Mode Trigger allows triggering the ICD device to move from the idle to active state and make it immediately responsive, for example to change its configuration.
```

  This option is by default enabled for the LIT device.

The LIT, CIP and UAT features were not finalized for Matter v1.3 and they are marked as provisional, so it is not recommended to use them, though you can find some of the LIT implementation in the Matter SDK and Matter specification.

> You can still enable them for the testing purposes.

Reviewer: ```suggestion
You can still enable them for testing purposes.
```
